src/frontend/src/components/alias/index.ts

```diff
@@ -11,7 +11,7 @@ import copyJson from "./index.json";
 
 export const promptDeviceAliasTemplate = (props: {
   title: string;
-  message?: string;
+  message?: string | TemplateResult;
```

Comment:
nth: title, message and cancelText are similar for all three props. a common interface could be extracted

Reply:
True, might do this separately if it becomes a pain